### ISSUE TITLE
Scanner room detecting already broken outcrops

### ISSUE THREAD

**Issue by BeefyAndTheDucks** [SubnauticaNitrox/Nitrox#2151]
### Are you on the latest version of Nitrox?

Yes, I'm on the latest commit of the repo (only for experts).

### Which OS are you playing on?

Windows

### On which platform did you buy Subnautica?

Steam

### Describe the issue.

So my friend mined some shale and when I scanned for shale outcrops the Scanner Room is still detecting those outcrops.

### Describe the steps you took to encounter the bug.

1. Get your friend to break some outcrops
2. Scan for that outcrop type
3. Watch as the scanner room finds already broken outcrops

### Relevant log output

Not required.

**Comment by oliver408i** [comment]
Scanner rooms aren't synced

**Comment by BeefyAndTheDucks** [comment]
Yeah I figured, Is that something that is currently being worked on/already has a PR?

**Comment by dartasen** [comment]
> Yeah I figured, Is that something that is currently being worked on/already has a PR?

It's not something that is being worked on currently

**Comment by gussandst** [comment]
Verified to be an issue.

**Comment by Gauvino** [comment]
Also still an issue
